torch/ao/quantization/quantizer/embedding_quantizer.py

```diff
@@ -12,18 +13,38 @@
     QuantizationSpec,
     Quantizer,
 )
-from torch.ao.quantization.quantizer.xnnpack_quantizer_utils import (
-    OperatorConfig,
-    OperatorPatternType,
-    QuantizationConfig,
-)
-
 
 __all__ = [
     "get_embedding_operators_config",
     "EmbeddingQuantizer",
 ]
 
+# In the absence of better name, just winging it with QuantizationConfig
+@dataclass(eq=True, frozen=True)
```

Comment:
These need to be added to the `__all__` block above and be properly documented on the doc website if you want to move them here as public API.